composer.json

```diff
@@ -1,10 +1,25 @@
 {
-    "require-dev": {
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
-        "wimg/php-compatibility": "dev-master",
-        "wp-coding-standards/wpcs": "dev-develop"
-    },
-    "scripts": {
-        "lint": "phpcs"
-    }
+  "name": "wordpress/gutenberg",
+  "type": "wordpress-plugin",
+  "license": "GPL-2.0",
+  "description": "Prototyping since 1440. Development hub for the editor focus in core.",
+  "homepage": "https://wordpress.github.io/gutenberg/",
+  "keywords": [
+    "gutenberg", "wordpress", "editor", "wp", "react", "javascript"
+  ],
+  "support": {
+    "issues": "https://github.com/WordPress/gutenberg/issues"
+  },
+  "require-dev": {
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
+    "squizlabs/php_codesniffer": "2.9.x",
+    "wimg/php-compatibility": "dev-master",
+    "wp-coding-standards/wpcs": "^0.14.0"
+  },
+  "require": {
+    "composer/installers": "~1.0"
+  },
+  "scripts": {
+    "lint": "phpcs"
```

Comment:
PHPCS does more than lint, and adding a script that then expects `composer lint`, instead of just the single command `phpcs` seems a little redundant.

Reply:
Totally agreed. I'll remove it.